src/plugins/navigation/server/config.ts

```diff
@@ -11,11 +11,11 @@ import type { PluginConfigDescriptor } from '@kbn/core-plugins-server';
 
 const configSchema = schema.object({
   solutionNavigation: schema.object({
-    featureOn: schema.boolean({ defaultValue: false }),
-    enabled: schema.boolean({ defaultValue: false }),
+    featureOn: schema.boolean({ defaultValue: true }),
```

Comment:
The changes in this file will be reverted.
I added them so the cloud deployment preview renders the new nav by default